code/mob/living/silicon/ai.dm

```diff
@@ -1380,7 +1380,12 @@ var/list/ai_emotions = list("Happy" = "ai_happy",\
 	for(var/datum/db_record/t as anything in data_core.general.records)
 		crew += "[t["name"]] - [t["rank"]]<br>"
 
-	usr.Browse("<head><title>Crew Manifest</title></head><body><tt><b>Crew Manifest:</b><hr>[crew]</tt></body>", "window=aimanifest")
+	var/obj/cryotron/cryo_unit = pick(by_type[/obj/cryotron])
```

Comment:
This will error out if there are zero cryotrons. Consider wrapping the relevant part in `if(length(by_type[/obj/cryotron]))`, just keep `var/stored = ""` outside of the if.

Reply:
Updated on remote; after testing it works as expected. No runtimes when I delete the cryotron and try to view any of the manifests. Since the cryotron should NEVER EVER be deleted anyways, the fact that the 'In Cryogenic Storage' list turns empty even when there are people who ghosted in cryo isn't a big deal.

Ran all the tests I mentioned in the original PR as well and I think I'm ready to commit. Just double checked to make sure I implemented all the requested changes (admittedly manually since I didn't notice the command to merge the requested change). *Hopefully* I have fucked nothing up!